src/raftstore/store/debug.rs

```diff
@@ -219,6 +220,31 @@ impl Debugger {
         compact_range(db, handle, start, end, false);
         Ok(())
     }
+
+    /// Set a region to tombstone by manual.
+    pub fn set_region_tombstone(
```

Comment:
why not using a meta::Region directly as the argument.

Reply:
How about the region info in PD is updated? For example, if the region split it will have a new [start_key, end_key).

Reply:
It's also weird the range doesn't match the epoch.